src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllProvider.cs

```diff
@@ -59,6 +59,38 @@ public static FixAllProvider Create(Func<FixAllContext, Document, ImmutableArray
             return new CallbackDocumentBasedFixAllProvider(fixAllAsync);
```

Comment:
consider having this method defer to the one below it.

Reply:
That method requires supported fix all scopes, and GetDefaultSupportedFixAllScopes is an instance method on FixAllProvider. `CallbackDocumentBasedFixAllProvider` has two ctors, which call into different base type ctors, so these are different code paths.